metricbeat/module/statsd/server/_meta/data.json

```diff
@@ -0,0 +1,19 @@
+{
+    "@timestamp":"2016-05-23T08:05:34.853Z",
+    "beat":{
+        "hostname":"beathost",
+        "name":"beathost"
+    },
+    "metricset":{
+        "host":"localhost",
+        "module":"http",
+        "name":"server",
```

Comment:
this file will need to be regenerated by running integration tests with -data flag (or manually if there is no TestData test)